actix-server/src/handlers/api_key_handler.rs

```diff
@@ -0,0 +1,48 @@
+use super::auth_handler::AuthedUser;
+use crate::{data::models::PgPool, operators::api_key_operator::create_api_key_query};
+use actix_web::{web, HttpResponse};
+use serde::{Deserialize, Serialize};
+use utoipa::ToSchema;
+
+#[derive(Debug, Serialize, Deserialize, ToSchema)]
+pub struct CreateApiKeyReqPayload {
+    /// The name which will be assigned to the new api key.
+    name: String,
+}
+
+#[derive(Serialize, Deserialize, ToSchema)]
+pub struct CreateApiKeyRespPayload {
+    /// The api key which was created. This is the value which should be used in the Authorization header.
+    api_key: String,
+}
```

Comment:
Can we change these type names so that they are more distinct

Reply:
Yeah, I'm game. What are you thinking?